infrastructure/kube/keep-test/electrs-statefulset.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: electrs
+  namespace: default
+  labels:
+    app: electrs
+    type: electrs
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: electrs
+      type: electrs
+  serviceName: electrs
+  volumeClaimTemplates:
+    - metadata:
+        name: electrs-data
+      spec:
+        accessModes: [ReadWriteOnce]
+        resources:
+          requests:
+            storage: 20Gi
+    - metadata:
+        name: electrs-config
+      spec:
+        accessModes: [ReadWriteOnce]
+        resources:
+          requests:
+            storage: 5Mi
+  template:
+    metadata:
+      labels:
+        app: electrs
+        type: electrs
+    spec:
+      initContainers:
+        - name: initcontainer-provision-electrs
+          image: gcr.io/keep-test-f3e0/initcontainer-provision-electrs:latest
+          imagePullPolicy: Always
+          env:
+            - name: JSON_RPC_IMPORT
+              value: 'true'
+            - name: BITCOIN_ADDRESS
+              valueFrom:
+                configMapKeyRef:
+                  name: electrs-btc-connectivity
+                  key: btc-address
+            - name: BITCOIN_USERNAME
+              valueFrom:
+                configMapKeyRef:
+                  name: electrs-btc-connectivity
+                  key: btc-user
+            - name: BITCOIN_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: bcoin
+                  key: http-api-key
```

Comment:
I'm wondering whether we can use just `BITCOIN_ADDRESS` and pass here the `bcoin-url` key from the `bcoin` secret. That key contains both user and password in the URL. If so, we can get rid of the `electrs-btc-connectivity` config map and we can avoid setting `BITCOIN_USERNAME` and `BITCOIN_PASSWORD` envs here. What do you thing? Will electrs handle this?

Reply:
`bcoin-url` contains username, password and address, so I could get rid of `BITCOIN_USERNAME`, `BITCOIN_PASSWORD` and `BITCOIN_ADDRESS`. It was done in https://github.com/keep-network/electrs/pull/1/commits/16a4bbab8eeb98381180639f96a9c4b666e40e42.